### PR TITLE
config workaround

### PR DESCRIPTION
**Pull request by senia-psm**

### CODE REVIEW

File: golem-shard-manager/src/shard_manager_config.rs [hunk 1]
```diff
@@ -61,11 +61,12 @@ pub struct HealthCheckConfig {
 #[derive(Clone, Debug, Deserialize)]
 #[serde(tag = "type", content = "config")]
 pub enum HealthCheckMode {
-    Grpc,
-    #[cfg(feature = "kubernetes")]
-    K8s {
-        namespace: String,
+    Grpc {
+        // Useless. Just to make config lib happy.
+        _dummy: Option<u8>,
```
Comment (justcoon) [inline golem-shard-manager/src/shard_manager_config.rs:66]: maybe better is to do `Grpc(golem_service_base::model::Empty)`, like here

https://github.com/golemcloud/golem/blob/main/golem-component-service-base/src/config.rs#L22

Comment (senia-psm) [reply golem-shard-manager/src/shard_manager_config.rs:66]: In this case I'll have to replace `K8s { namespace: String }` with `K8s(K8sMode)`. I'm not sure it worth the refactoring.
WDYT?

Comment (justcoon) [reply golem-shard-manager/src/shard_manager_config.rs:66]: not, sure, maybe we can do later ...

**Review by justcoon — APPROVED** [review]

File: golem-shard-manager/src/shard_manager_config.rs (hunk 1 above)
Comment (senia-psm) [reply golem-shard-manager/src/shard_manager_config.rs]: Done. It was not that bad as I expected.

**Review by vigoo — APPROVED** [review]

**Pull request merged**